llm_on_ray/inference/serve.py

```diff
@@ -179,15 +185,25 @@ def main(argv=None):
         host = "127.0.0.1" if args.serve_local_only else "0.0.0.0"
         print("Service is running with deployments:" + str(deployments))
         print("Service is running models:" + str(model_list))
-        openai_serve_run(
-            deployments,
-            model_list,
-            host,
-            "/",
-            args.port,
-            args.max_ongoing_requests,
-            args.max_num_seqs,
-        )
+
+        if args.openai_route_prefix:
+            openai_serve_run(
+              deployments, 
+              model_list, 
+              host, 
+              "/" + args.openai_route_prefix, 
+              args.port, 
+              args.max_ongoing_requests, 
+              args.max_num_seqs,)
+        else:    
+            openai_serve_run(
+              deployments, 
+              model_list, 
+              host, 
+              "/", 
+              args.port, 
+              args.max_ongoing_requests, 
+              args.max_num_seqs,
```

Comment:
Then we don't need this judgment, just pass in the parameter `openai_route_prefix`

Reply:
Thanks for the input, implementing it in following commit.